KakaoImageSearch/Api/Model/ImageItem.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ImageItem.swift
+//  KakaoImageSearch
+//
+//  Created by 강현석 on 2021/01/06.
+//
+
+import Foundation
+
+struct ImageItem: Codable {
```

Comment:
P3: 저는 개인적으로 코드의 범위를 최소화하는 것을 좋아합니다.
해서 해당 모델을 Encoding 할 일이 없다면, Codable 말고 Decodable만 선언해줘도 좋습니다.
어차피 Codable = Encodable & Decodable이라서요
```suggestion
struct ImageItem: Decodable {
```

Reply:
꿀팁 감사합니다 👍